microsoft-edge/extensions-chromium/publish/api/using-addons-publish-api.md

```diff
@@ -0,0 +1,220 @@
+
+The examples below use the domain `https://addons.edge.microsoft.com/api`, which is a placeholder and needs to be replaced by another example, given that this API is not live yet.
+
+
+<!-- ====================================================================== -->
+## Getting the list of products
+
+<!-- no section in Part 1 of spec -->
+
+API reference: [Get the list of products](addons-publish-api-reference.md#get-the-list-of-products)
+
+
+<!-- ====================================================================== -->
+## Uploading a package to update an existing submission
+
+Use this API to update the package for an add-on.  This API uploads a package to update an existing draft submission of an add-on product.
+
+```rest-api
+Endpoint: /v1/products/$productID/submissions/draft/package
+Type: PUT
+Header Parameters: $TOKEN: the access token; Content-Type: application/zip
+Body content: the package file to upload
+```
+
+`$productID` is the ID of the Microsoft Edge Add-on that you want to update.  You can get the product ID in either of the following ways:
+
+*  Login to Microsoft Partner Center.  Navigate to **Microsoft Edge > Overview**, and then select the extension you want the product ID for.  The extension overview page opens.  The GUID in the URL is the product ID.
+
+*  Call the `/products` API to get a list of all products and their product IDs.  For more information about the `/products` API, navigate to the [Add-ons Publish API Reference][AddonsPublishAPIRef].
+
+### Sample request
+
+```console
+> curl \
+-H "Authorization: Bearer $TOKEN" \
+-H "Content-Type: application/zip" \
+-X PUT \
+-T $FILE_NAME \
+-v \
+https://api.partnercenter.microsoft.com/edgeaddons/v1/products/$productID/submissions/draft/package
+```
+
+API reference: [Upload a package to update an existing submission](addons-publish-api-reference.md#upload-a-package-to-update-an-existing-submission)
+
+
+<!-- ====================================================================== -->
+## Checking the status of a package upload
+
+Use this API to check the status of package upload.
+
+```rest-api
+Endpoint: /v1/products/$productID/submissions/draft/package/operations/$operationID
+Type: GET
+Header Parameters: $TOKEN: the access token
+```
+
+### Sample request
+
+```console
+> curl \
+-H "Authorization: Bearer $TOKEN" \
+-X GET \
+-v \
+https://addons.edge.microsoft.com/api/v1/products/$productID/submissions/draft/package/operations/$operationID
+```
+
+API reference: [Check the status of a package upload](addons-publish-api-reference.md#check-the-status-of-a-package-upload)
+
+
+<!-- ====================================================================== -->
+## Publishing the product draft submission
+
+Use this API to publish the current draft of the product to the Edge Add-ons website.
+
+`https://addons.edge.microsoft.com/api` is a placeholder and needs to be replaced by another example.
+
+```rest-api
```

Comment:
global: rest-api didn't resolve even though listed at a site listing valid values.  could keep as-is (its lowercasing is hardly noticable), or find an alternative.  Symptoms: lowercase, no code coloring.

Reply:
Suggest a change here. 

`https://addons.edge.microsoft.com/api` is a placeholder and could be replaced once the API's are available in production.